package.json

```diff
@@ -2,22 +2,24 @@
   "name": "redux-persist",
   "version": "5.6.9",
   "description": "persist and rehydrate redux stores",
-  "main": "lib/index.js",
-  "module": "es/index.js",
+  "main": "dist/redux-persist",
```

Comment:
this is matching first draft of the configs - the ones that are commented out now